src/batResult.mli

```diff
@@ -38,6 +38,16 @@ val get : ('a, exn) t -> 'a
 *)
 val default: 'a -> ('a, _) t -> 'a
 
+(** [map f (Ok x)] returns [Ok (f x)] and [map f (Bad e)] returns [Bad e].
+    @since 2.5.4
+*)
+val map : ('a -> 'b) -> ('a, 'c) t -> ('b, 'c) t
+
+(** [map_both f g (Ok x)] returns [Ok (f x)] and [map_both f g (Bad e)] returns [Bad (g e)].
+    @since 2.5.4
```

Comment:
could you use `NEXT_RELEASE` instead of `2.5.4`, so that we can replace with the correct version (which may not be 2.5.4) at release time?